### DISCUSSION TITLE
when i click the tab2 ,it always 500 error

### DISCUSSION THREAD

**Discussion by shougua** [symfony/symfony#44326] [Q&A]
I want to change the style, but it always 500 https error or connect timeout. click design icon can  appear inevitably.

An error is as follows:
![微信图片_20211129161213](https://user-images.githubusercontent.com/31030240/143831051-7ae4ca60-8d00-482a-9d59-456d2e646306.png)

**Comment by stof** [comment]
This error is not happening in Symfony. You reported it to the wrong repo.
